data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -205,10 +205,13 @@ def _pollution_factor(self) -> bool:
 
         pollution_criteria = (
             self.df[field_names.RMP_FIELD + field_names.PERCENTILE_FIELD_SUFFIX]
-            > self.ENVIRONMENTAL_BURDEN_THRESHOLD
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
```

Comment:
This change from `>` to `>=` is fine, but if we do that, we should do it on all the criteria throughout Score L. Do you mind doing that? In practice I don't think it'll matter much because there are rarely exact ties but we should be consistent.

Reply:
@lucasmbrown-usds I am adhering to what @BethMattern stated in the issue. Your logic for across-the-board changes is fine - I just wanted to bring that to your attention.

Reply:
Sounds good. Can you go ahead and make this change everywhere then in Score L, for consistency?

Reply:
Yup